vercel.json

```diff
@@ -987,7 +987,8 @@
             "destination": "/docs/data/events#pageview-events"
         },
         { "source": "/docs/data/data-warehouse", "destination": "/docs/data-warehouse" },
-        { "source": "/docs/apps/zendesk-connector", "destination": "/docs/cdp/zendesk-connector" }
+        { "source": "/docs/apps/zendesk-connector", "destination": "/docs/cdp/zendesk-connector" },
+        { "source": "/blog/how-to-product-market-fit", "destination": "/blog/product-market-fit-game" }
```

Comment:
add`/blog/posthog-restack-announcement` to go somewhere? Idk where.

Reply:
Honestly, it's ok if it 404s. The trick is making sure there aren't any links to it!